.changelog/unreleased/breaking-changes/736-proto-enum-rename.md

```diff
@@ -0,0 +1,6 @@
+- `[abci]` Change the proto-derived enum type and constant aliases to the
+  buf-recommended naming conventions adopted in the `abci/v1beta4` proto package
```

Comment:
This line is outdated too, there is no longer `abci/v1beta4`.
I will fix this and above in a small PR.

Reply:
Fixed in #1707